libs/vm-modules/src/math/bignumber.cpp

```diff
@@ -105,11 +94,13 @@ UInt256Wrapper::UInt256Wrapper(VM *vm, TypeId type_id, uint64_t data)
 {}
 
 Ptr<UInt256Wrapper> UInt256Wrapper::ConstructorFromBytes(VM *vm, TypeId type_id,
-                                                         Ptr<ByteArrayWrapper> const &ba)
+                                                         Ptr<ByteArrayWrapper> const &ba,
+                                                         bool input_is_little_endian)
```

Comment:
Presumably it just makes sense to standardise this interface to always expecting big endian